submissions/team_api.py

```diff
@@ -101,7 +101,7 @@ def create_submission_for_team(
         model_kwargs["submitted_at"] = submitted_at
 
     try:
-        team_submission_serializer = TeamSubmissionSerializer(data=model_kwargs)
+        team_submission_serializer = TeamSubmissionSerializer(data=model_kwargs, context={'answer': answer})
```

Comment:
I think, for consistency, this should follow the `api.py` version,  where answer is part of the original `model_kwargs` unless there's a specific reason to break the pattern. It should result in fewer changes.

Reply:
there's no answer field on TeamSubmission, so i think that would raise an exception

Reply:
Ooh, my bad. I was thinking that `TeamSubmission` inherited from `Submission`. Carry on!